### DISCUSSION TITLE
Homer Web Interface HTTPS

### DISCUSSION THREAD

**Discussion by henristema** [sipcapture/homer#646] [Q&A] [answered]
Hi everyone,

I have deployed homer 7.0 using docker and everything is working fine.

At the moment I can access the web interface through my browser by http://127.0.0.1:9080.
Is there a way to use https instead of http, so it could be then accessed through https://127.0.0.1:9080.

I looked online, but I did not find anything, so any help is appreciated.

**Comment by lmangani** [comment]
Managing certificates directly on the service is possible but can be tedious and risky.
The best practice is not to expose the setup directly but to deploy an HTTPS proxy w/ certificates in front of your Homer setup

**Comment by adubovikov** [comment, marked as answer]
another way is to activate https ? 

```
"https_settings": {
        "help": "SSL settings for homer-app",
        "enable": true,
        "host": "0.0.0.0",
        "port": 9080,
        "min_tls_version": "1.2",
        "use_mozilla_profile": true,
        "cert": "/usr/local/homer/tls/cert.pem",
        "key": "/usr/local/homer/tls/key.pem"
    },

```
and set http to different port... 8080

here is the example config:
https://github.com/sipcapture/homer-app/blob/master/etc/webapp_config.json#L90-L99

**Reply by henristema** [reply]
Thank you for your reply!
